packages/studio/src/components/HighlightingContainer.tsx

```diff
@@ -87,6 +104,20 @@ class HighlightingClass extends Component<HighlightingProps> {
   }
 
   componentDidUpdate(): void {
+    const childNode = getDOMNode(this);
+    if (!childNode) {
+      return;
+    }
+    const rect = rectToJson(childNode.getBoundingClientRect());
+    const isRectUpdated =
+      JSON.stringify(this.props.selectedComponentRectsMap[this.props.uuid]) ===
+      JSON.stringify(rect);
+    if (
+      !isRectUpdated &&
```

Comment:
maybe I'm not following correctly, but shouldn't we be checking that `isRectUpdated` is truthy, not falsy?

Reply:
`isRectUpdated` was previously checking for equality between the rectangles rather than a change, so it was working properly but the naming was backwards. I moved the `!` to make the variable name more correct.